frontend/components/Tables.tsx

```diff
@@ -65,6 +65,28 @@ export function ConformanceTestRow(props: {
 
   const cellClasses = ['p-4', 'border-x-white', 'border-x-2'];
 
+  let techSpecs = '';
+  switch (pathfinder_version) {
+    case '1.0.0':
+      techSpecs =
+        'https://www.carbon-transparency.com/media/1qcdbdyn/pathfinder-network_technical-specifications-for-use-case-001.pdf';
+      break;
+    case '1.0.1':
+      techSpecs =
+        'https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230201/index.html';
+      break;
+    case '2.0.0':
+      techSpecs =
+        'https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/index.html';
+      break;
+    case '2.0.1':
+      techSpecs =
+        'https://wbcsd.github.io/tr/2023/data-exchange-protocol-20231109/index.html';
+      break;
+    case '2.1.0':
+      techSpecs = 'https://wbcsd.github.io/tr/data-exchange-protocol';
```

Comment:
```suggestion
      techSpecs = 'https://wbcsd.github.io/tr/2023/data-exchange-protocol-20231207/';
```

Please use this link which is a permanent one specific to this version.